i18n/en-US.yml

```diff
@@ -707,15 +712,15 @@ common:
     bus: Bus
     # The original OTP mode id is CABLE_CAR. Lowercase makes it cable_car.
     cable_car: Cable Car
-    car: Car
     car_park: Park and Ride
+    car: Car
```

Comment:
I think car should be sorted before car_park, but that's pretty nit picky.

Reply:
I appreciate nit-picky-ness!